moped-editor/src/components/GridTable/Pagination.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { TablePagination } from "@mui/material";
+
+/**
+ * Pagination Component
+ * @param {Number} recordCount - total number of records
+ * @param {Number} queryLimit - limit of records per page
+ * @param {Function} setQueryLimit - set the limit of records per page
+ * @param {Number} offset - offset
+ * @param {Function} setOffset - set offset
+ * @param {Array} rowsPerPageOptions - array of numbers for rows per page
+ * @return {JSX.Element}
+ * @constructor
+ */
+const Pagination = ({
+  recordCount,
+  queryLimit,
+  setQueryLimit,
+  queryOffset,
+  setQueryOffset,
+  rowsPerPageOptions,
+}) => {
+  /**
+   * Handles Page Change (Next or Previous)
+   * @param {Object} event - The object with event details
+   * @param {integer} newPage - The actual page number as an integer
+   */
+  const handleChangePage = (event, newPage) => {
+    // Calculate new offset
+    const newOffset = newPage * queryLimit;
+    // Update State
+    setQueryOffset(newOffset);
+  };
+
+  /**
+   * Handles the "Rows Per Page" change in the drop down menu
+   * @param {Object} event - The object with event details
+   */
+  const handleChangeRowsPerPage = (event) => {
+    // Transform the RowsPerPage number from string to integer, call it newLimit
```

Comment:
oooh does this fix that warning that shows up about invalid options??

Reply:
Not sure - I don't think that I've noticed that warning before, and I can't make it happen. Maybe a happy accident? Let me know if you see it happen.

Reply:
![Screenshot 2023-09-29 at 9 18 42 AM](https://github.com/cityofaustin/atd-moped/assets/12474808/651d0070-f9f7-4f89-8265-df0035d9d527)
This is what i see in main, and i don't see it in this PR, so i do think happy accident!